.github/workflows/jazzy-build-win.yml

```diff
@@ -0,0 +1,29 @@
+name: Windows Binary Build
+# author: Christoph Fröhlich <christoph.froehlich@ait.ac.at>
+# description: 'Build & test all dependencies from released (binary) windows packages.'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    # types:
+    #   - labeled
+  push:
+    branches:
+      - master
+  # issue_comment:
+  #   types:
+  #     - created
+
+jobs:
+  binary-windows:
+    # if: |
+    #   (github.event_name == 'issue_comment' && contains(github.event.comment.body, '/check-windows')) ||
+    #   (github.event_name == 'pull_request' && contains(github.event.label.name, 'check-windows')) ||
+    #   (github.event_name == 'workflow_dispatch')
```

Comment:
Ah sorry, I haven't added the reference to https://github.com/ros-controls/realtime_tools/pull/204
I was just adding these comments to get reminded of this possibility in case the windows build do not work well, or take way more time then all the other jobs.